### PR TITLE
Feature/166 ci cypress tests inclusion

### PR DESCRIPTION
**Pull request by muganwas**

### CODE REVIEW

**Comment by choshun** [comment]
Awesome, thx so much for cleaning this up!!

File: cypress.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-  "defaultCommandTimeout": 15000,
-  "viewportWidth": 1300,
-  "reporter": "junit",
-  "reporterOptions": {
-    "mochaFile": "cypress/results/cypress-report-[hash].xml"
-  }
-}
\ No newline at end of file
+	"chromeWebSecurity": false,
+	"baseUrl": "https://roundaround-stage.web.app/",
```
Comment (choshun) [inline cypress.json:3]: So for this PR the tests will always point to stage?

**Comment by choshun** [comment]
Do you mind saying what part of the ticket is addressing? Know you mentioned a lot of this work was actually in github

File: cypress.json (hunk 1 above)
Comment (muganwas) [reply cypress.json:3]: yes, but we can change it on a per branch basis. Also the local test use localhost, I set that up in package.json

**Comment by muganwas** [comment]
> Do you mind saying what part of the ticket this is addressing? Maybe in the description of the PR? Know you mentioned a lot of this work was actually in github

This added the yml file that has action configurations. Forgot to edit the title.

**Comment by choshun** [comment]
Yah, see you added the yml file after the PR was created, thx

Guess I'll try one more time, what part of this ticket: https://github.com/irllabs/roundaround/issues/166

Is left after this PR?

**Comment by choshun** [comment]
This PR basically does the first out of 3 here:
https://github.com/irllabs/roundaround/issues/166

**Review by choshun — APPROVED** [review]
Thanks!

**Pull request merged**